tests/unittests/evaluation_tests.py

```
@@ -3381,6 +3709,145 @@ def test_evaluate_video_segmentations_simple(self):
+                ]
+            )
+
+            cls.dataset.add_sample(sample)
+
+    @classmethod
+    @drop_datasets
+    def tearDownClass(cls):
+        try:
+            if cls.dataset.name in fo.list_datasets():
+                cls.dataset.delete()
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)